PSL/packet_handler.py

```diff
@@ -1,231 +1,303 @@
-from __future__ import print_function
+"""Low-level communication for PSLab.
 
+Example
+-------
+>>> from PSL.packet_handler import Handler
+>>> H = Handler()
+>>> version = H.get_version()
+>>> H.disconnect()
+"""
+from functools import partial
+import logging
+import struct
 import time
+from typing import List, Union
 
 import serial
-
-# Handle namespace conflict between packages 'pyserial' and 'serial'.
```

Comment:
I ended up removing this again, because `from serial.tools import list_ports` will raise an ImportError if the wrong `serial` is installed.